src/query/tsdb/remote/codecs.go

```diff
@@ -190,23 +138,62 @@ func encodeTagMatchers(modelMatchers models.Matchers) (*rpc.TagMatchers, error)
 	}, nil
 }
 
+func encodeFanoutOption(opt storage.FanoutOption) (rpc.FanoutOption, error) {
+	switch opt {
+	case storage.FanoutDefault:
+		return rpc.FanoutOption_DEFAULT_OPTION, nil
+	case storage.FanoutForceDisable:
+		return rpc.FanoutOption_FORCE_DISABLED, nil
+	case storage.FanoutForceEnable:
+		return rpc.FanoutOption_FORCE_ENABLED, nil
+	}
+
+	return rpc.FanoutOption_FORCE_ENABLED,
+		fmt.Errorf("Unknown fanout option for proto encoding: %v\n", opt)
```

Comment:
Hm seems this one could be `return 0, fmt.Errorf("unknown ...")` as pinged about elsewhere